src/test/java/edu/ucsb/cs156/happiercows/jobs/UpdateCowHealthJobTests.java

```diff
@@ -1,33 +1,355 @@
 package edu.ucsb.cs156.happiercows.jobs;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Answers.valueOf;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.ArgumentMatchers.longThat;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
 
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
 
+import edu.ucsb.cs156.happiercows.entities.Commons;
+import edu.ucsb.cs156.happiercows.entities.User;
+import edu.ucsb.cs156.happiercows.entities.UserCommons;
 import edu.ucsb.cs156.happiercows.entities.jobs.Job;
+import edu.ucsb.cs156.happiercows.repositories.CommonsRepository;
+import edu.ucsb.cs156.happiercows.repositories.UserCommonsRepository;
 import edu.ucsb.cs156.happiercows.services.jobs.JobContext;
 
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration
 public class UpdateCowHealthJobTests {
+
+
+    @MockBean 
+    CommonsRepository commonsRepository;
+    
+    @MockBean
+    UserCommonsRepository userCommonsRepository;
+
     @Test
     void test_log_output() throws Exception {
 
         // Arrange
 
+        List<Commons> commonsList = new ArrayList<Commons>();
+
+        when(commonsRepository.findAll()).thenReturn(commonsList);
+
+
         Job jobStarted = Job.builder().build();
         JobContext ctx = new JobContext(null, jobStarted);
 
         // Act
-        UpdateCowHealthJob updateCowHealthJob = UpdateCowHealthJob.builder()
-                .build();
+
+        UpdateCowHealthJob updateCowHealthJob = new UpdateCowHealthJob(commonsRepository, userCommonsRepository);
         updateCowHealthJob.accept(ctx);
 
         // Assert
 
         String expected = "Starting to update cow health.\n" +
-                "This is where the code to update the cow's health will go.\n" +
+               // "This is where the code to update the cow's health will go.\n" +
```

Comment:
We don't leave commented out code in PRs that are intended to be merged into main.

Please don't do this when you end up at an internship or job.  

And to whomever code reviewed this: you should have caught that.

Reply:
Commented out code has been removed.